src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralTest.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import java.text.SimpleDateFormat
+import java.util.UUID
+
+class DraftReferralTest : IntegrationTestBase() {
+
+  @Test
+  fun `create draft referral dto from empty referral`() {
+    assertThrows<RuntimeException> {
+      DraftReferral(Referral())
+    }
+  }
+
+  @Test
+  fun `create draft referral dto with empty referral id`() {
+    val createdDate = SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse("2020-12-04 10:42:43")
+
+    assertThrows<RuntimeException> {
+      DraftReferral(Referral(created = createdDate))
```

Comment:
Would it be worth creating an "example valid" referral and setting its `id` to `null`? It may make the intent in the test description clearer (and not depend on remembering to add the other fields when they're added).

Reply:
At the moment, only 2 fields are mandatory for a valid Referral.  I think this seems fine.   If this changes, I can update the test to reflect this (as you suggested).